driver/swift.go

```diff
@@ -58,9 +58,14 @@ func NewSwiftDriver(source *models.Source) (Driver, error) {
 		return nil, fmt.Errorf("Unable to get container by name '%s', inner error: %s", source.OpenStack.Container, err.Error())
 	}
 
-	initialVersion, err := semver.Parse(source.InitialVersion)
-	if err != nil {
-		return nil, fmt.Errorf("Initial version was not a valid sem ver: %s", err.Error())
+	var initialVersion semver.Version
```

Comment:
this logic looks right, matches https://github.com/concourse/semver-resource/blob/master/driver/driver.go#L27-L36